### PR TITLE
Add event for GDCR Singapore 2020

### PR DESCRIPTION
**Pull request by miccheng**
On 7th November. Virtual event.

### CODE REVIEW

File: _data/events/virtual-2020-11-06-global-day-of-coderetreat-singapore-2020.json [hunk 1]
```diff
@@ -0,0 +1,31 @@
+{
+  "title": "Global Day of CodeRetreat Singapore 2020",
+  "format": "classic",
+  "spoken_language": "English",
+  "url": "https://www.meetup.com/Junior-Developers-Singapore/events/274176954",
+  "date": {
```
Comment (rradczewski) [inline _data/events/virtual-2020-11-06-global-day-of-coderetreat-singapore-2020.json:6]: Your meetup event is for the 7th, this payload is for the 6th. Could you fix this please before we merge it?

Comment (miccheng) [reply _data/events/virtual-2020-11-06-global-day-of-coderetreat-singapore-2020.json:6]: Sure. Updated. Thanks!

**Comment by rradczewski** [comment]
Excellent, welcome and thanks for submitting your event!

**Comment by rradczewski** [comment]
Hi, we hope you had a great Global Day Of Coderetreat! We're trying to understand how the current situation has affected the events this year and how we could support you next year for #GDCR2021.

For this purpose, we created [a seven-questions-survey](https://docs.google.com/forms/d/e/1FAIpQLSdi8fmw42s8tXtVneJH5VcRpB9vMf35gJ2APw4PijQ0-Gq_Ww/viewform?usp=sf_link) (anonymously, all optional, single page) for those who organized an event this year. It would help us tremendously if you could fill it out. It doesn't take long, promise :smile: 

Thanks and all the best,
Raimo

**Pull request merged**